umm-spec-lib/test/cmr/umm_spec/test/migration/version_migration.clj

```diff
@@ -1129,3 +1131,51 @@
             {:URLs ["www.contact.shoo.com"],
              :Description "Contact related url description"}]
            collection-contact-persons))))
+
+(deftest migrate-1_9-up-to-1_10
+  (is (= u/NOT-PROVIDED
+        (:CollectionProgress
+          (vm/migrate-umm {} :collection "1.9" "1.10"
```

Comment:
I don't understand this part of the test, its the only one that doesn't match the mapping.